tika-core/src/main/resources/org/apache/tika/mime/tika-mimetypes.xml

```diff
@@ -6422,7 +6422,7 @@
       <!-- match X- DKIM- ARC- at start of file and then require at least one
            of the usual: from, received, date...but look farther into the file
            because of the X|DKIM|ARC headers-->
-      <match value="(X|DKIM|ARC)-" type="regex" offset="0">
+      <match value="(X|DKIM|ARC)-" type="regex" offset="0:1024">
```

Comment:
I worry about looking for X- anywhere in the first 1024 without requiring a \n before it.

What would you think of adding something like this into the previous minShouldMatch=2 clause?
`
<match value="\nX-" type="string" offset="0:1024">
<match value="\nDKIM-" type="string" offset="0:1024">
<match value="\nARC-" type="string" offset="0:1024">
`

Reply:
Good point, I like your solution better. Code changed accordingly.